amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/rm/rest/api/impl/SearchTypesFactory.java

```diff
@@ -92,7 +92,7 @@ public Set<QName> buildSearchTypesForUnfiledEndpoint(Parameters parameters, Set<
         boolean includeRecords = false;
```

Comment:
By default boolean value will be false. Should we need to assign values.

Reply:
Agrees but this is existing code.